bin/setup

```diff
@@ -20,11 +20,6 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  puts "\n== Copying sample files =="
```

Comment:
How is this related to Yarn and accessible autocomplete?

Reply:
This PR came from when I first started working on the project, so it partly turned into improving the README/setup so that it's easier. I've never run bin/setup on a rails project before and thought it was hiding a bit too much magic that should be in the README.